web/src/hotkey.js

```diff
@@ -788,6 +796,16 @@ export function process_hotkey(e, hotkey) {
         return true;
     }
 
+    if (event_name === "toggle_edit_preview" && compose_state.composing()) {
+        e.preventDefault();
+
+        if ($("#compose .markdown_preview").css("display") !== "none") {
```

Comment:
Can we instead use `$("#preview_message_area").is(":visible")` here? We use this in one more place in this file itself.